### PR TITLE
Add branching for st2tests repo

### PR DESCRIPTION
**Pull request by humblearner**

### CODE REVIEW

**Comment by m4dcoder** [comment]
LGTM

File: actions/setup_e2e_tests.sh [hunk 1]
```diff
@@ -30,11 +32,23 @@ sudo bash -c "cat <<keyvalue_options >>${ST2_CONF}
 encryption_key_path=${CRYPTO_KEY_FILE}
 keyvalue_options"
 
-st2 run packs.install subtree=true repo_url=StackStorm/st2tests packs=tests,asserts,fixtures,webui
+if [[ ${VERSION} == "master" ]]; then
+    echo "Installing st2tests from '${VERSION}' branch at location: `pwd`..."
+    git clone -b ${VERSION} --depth 1 https://github.com/StackStorm/st2tests.git
```
Comment (lakshmi-kannan) [inline actions/setup_e2e_tests.sh]: Why do we need this git clone and st2 run packs.install in line 39?

File: actions/setup_e2e_tests.meta.yaml [hunk 2]
```diff
@@ -5,6 +5,12 @@ enabled: true
 runner_type: remote-shell-script
 entry_point: setup_e2e_tests.sh
 parameters:
+  version:
```
Comment (lakshmi-kannan) [inline actions/setup_e2e_tests.meta.yaml]: Should be called branch to avoid confusions

File: actions/setup_e2e_tests.sh [hunk 3]
```diff
@@ -30,11 +32,23 @@ sudo bash -c "cat <<keyvalue_options >>${ST2_CONF}
...
+    echo "Installing Packs: tests, asserts, fixtures, webui..."
+    st2 run packs.install subtree=true branch=${VERSION} repo_url=StackStorm/st2tests packs=tests,asserts,fixtures,webui
+else
+   echo "Installing st2tests from 'v${VERSION}' branch at location: `pwd`"
+   git clone -b v${VERSION} --depth 1 https://github.com/StackStorm/st2tests.git
```
Comment (lakshmi-kannan) [inline actions/setup_e2e_tests.sh]: What if someone supplies 2.0.1 as version? Our branches are v2.0 etc.

**Review by lakshmi-kannan — CHANGES_REQUESTED** [review]

File: actions/setup_e2e_tests.sh (hunk 1 above)
Comment (humblearner) [reply actions/setup_e2e_tests.sh]: git clone is for robotfm tests and st2 run packs.install is for all the other packs. Well we need to make robotfm pack eventually :)

File: actions/setup_e2e_tests.meta.yaml (hunk 2 above)
Comment (humblearner) [inline actions/setup_e2e_tests.meta.yaml]: done!!

File: actions/setup_e2e_tests.sh (hunk 3 above)
Comment (humblearner) [reply actions/setup_e2e_tests.sh]: https://github.com/StackStorm/st2cd/pull/211/files#diff-b585755378dcc390127e7d68b1360256R5

**Review by lakshmi-kannan — APPROVED** [review]

**Comment by lakshmi-kannan** [comment]
LGTM. Merge and deploy and see things 🔥

**Pull request merged**